src/material/list/testing/list-harness-filters.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {BaseHarnessFilters} from '@angular/cdk/testing';
+
+export interface ListHarnessFilters extends BaseHarnessFilters {}
+
+export interface ActionListHarnessFilters extends BaseHarnessFilters {}
+
+export interface NavListHarnessFilters extends BaseHarnessFilters {}
+
+export interface SelectionListHarnessFilters extends BaseHarnessFilters {}
+
+export interface BaseListItemHarnessFilters extends BaseHarnessFilters {
+  text?: string | RegExp;
+}
+
+export interface ListItemHarnessFilters extends BaseListItemHarnessFilters {}
```

Comment:
Any reason not to just have `ListItemHarnessFilters` be the base type instead of having a separate `BaseListItemHarnessFilters` type?

Reply:
It didn't feel right since the other types don't actually extend MatList. However, I don't really see needing to add a filter for normal list items that doesn't also apply to the other list item types, so it would probably be fine if we want to do that.